how to define functions

```diff
@@ -0,0 +1,62 @@
+vals = (3, 7)
+
+f x = (x, x + 1)
+
+g :: Int -> (Int, Int)
+g x = (x+1, x)
```

Comment:
Hi, I was wondering if you could clarify your intent in transposing the tuple elements between `f` and `g`. We have some guesses based on how you use them below, but we'd like to make sure we understand your goal. Thank you.

Reply:
Hi! I didn't have any particular goal, I just thought it would be better to have functions slightly different.